### PR TITLE
scripts/get_touchpad_info.sh: add srcipt for dumping TP info

### PR DESCRIPTION
**Pull request by mkopec**
Signed-off-by: Michał Kopeć <michal.kopec@3mdeb.com>

### CODE REVIEW

File: scripts/get_touchpad_info.sh [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+	echo "This script must be run as root!"
+	exit
+fi
+
+echo "Kernel log:"
+dmesg
+
+echo "Installing dependencies"
+apt-get update
+apt-get install -y i2c-tools acpi-call-dkms
```
Comment (miczyg1) [inline scripts/get_touchpad_info.sh]: nope, this won't work. Add these to the Dockerfile

File: scripts/get_touchpad_info.sh [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+echo "running i2cdetect (quick write mode)"
+i2cdetect -y -q 1
+
+echo "running i2cdetect (receive byte mode)"
+i2cdetect -y -r 1
+
+echo "Loading module acpi_call"
+modprobe acpi_call
+
+devpath=$(dmesg | grep hid-generic | awk '{print $4}' | cut -d ':' -f 1-3)
```
Comment (miczyg1) [inline scripts/get_touchpad_info.sh]: This also won't work as expected. dmesg from my PC:

```
[    1.512777] hid-generic 0003:413C:2113.0001: input,hidraw0: USB HID v1.11 Keyboard [Dell KB216 Wired Keyboard] on usb-0000:00:1a.0-1.1/input0
[    1.572904] hid-generic 0003:413C:2113.0002: input,hidraw1: USB HID v1.11 Device [Dell KB216 Wired Keyboard] on usb-0000:00:1a.0-1.1/input1
[    1.631587] hid-generic 0003:413C:301A.0003: input,hidraw2: USB HID v1.11 Mouse [PixArt Dell MS116 USB Optical Mouse] on usb-0000:00:1a.0-1.2/input0
```

We surely are not interested in USB HID devices...

**Review by miczyg1 — CHANGES_REQUESTED** [review]
Please at least build a container, add the touchpad detection script call to the getlogs.sh and test it on HW

File: scripts/get_touchpad_info.sh (hunk 2 above)
Comment (mkopec) [reply scripts/get_touchpad_info.sh]: right, added an extra grep on `I2C HID`. Also now the script exits if no i2c hid is found in dmesg

**Comment by macpijan** [comment]
@miczyg1 

> Please at least build a container, add the touchpad detection script call to the getlogs.sh and test it on HW

This is tricky, as:
- we need access to i2c devices, which we currently do not have access to when running this container
- the `acpi-call-dkms` is a kernel module, which means it must be loaded on the host and match given kernel version - cannot be easily distributed inside of the container

I think we just dumped this script here for reference, but we may not be integrating it into the container, especially given that we want to replace it with different tools.

**Comment by pietrushnic** [comment]
@miczyg1 @macpijan the reason to create DTS OS is avoiding container complexity. We should stop use containerization and solely rely on iPXE or USB booted DTS OS. The same thing should happen in validation.

**Comment by macpijan** [comment]
agree

**Comment by macpijan** [comment]
Moved as a DTS feature.

**Pull request closed without merging**